modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -58,6 +58,17 @@ describe('browser.tinymce.core.content.EditorContentTest', () => {
     assert.isTrue(isExpectedTypes);
   };
 
+  const isSafari = () => PlatformDetection.detect().browser.isSafari();
+
+  const testGetTextContent = (content: string, expected: string) => {
+    const editor = hook.editor();
+    editor.setContent(content);
+    const text = editor.getContent({ format: 'text' });
+    Assertions.assertEq('Should be expected text', expected, text);
```

Comment:
This is a legacy API that shouldn't be used anymore (only the HTML functions should be used in `Assertions`). Please use the chai assertion library instead.

- [ ] Use chai assertions here instead of the legacy custom assertion API